src/ActionModal/components/ActionModal.vue

```diff
@@ -499,8 +498,11 @@ export default {
     selectedBalance() {
       const defaultBalance = {
         amount: 0,
-        // awful network-specific logic. But how to do it otherwise?
-        gasPrice: gasPricesDictionary(this.getDenom)
+        gasPrice:
```

Comment:
ur replicating the same logic as lines 501+
why is line 511 not delivering a balance is the question here

Reply:
Yes, you were right. The problem was exactly there, why those lines were not working.

And I found out. It was because this strange thing doing `({ denom: balanceDenom }) => balanceDenom === denom`.

Much more simple to rename the const `denom` and have it more readable (and working). Plus `balanceDenom` wasn't being used anywhere else.